tutorial/openEO_intro/Agenda_and_Links.md

```diff
@@ -0,0 +1,42 @@
+# Part-2: OpenEO (Comment: Intro Session)
+
+This Part is going to introduce participants to openEO Platform. Attendees will learn what openEO Platform is, how to find data and run first process graphs. 
+We will also show attendees how to integrate custom code into your workflow using User Defined Functions.
+
+## 11:00 Getting started with OpenEO (15 minutes) 
+Benjamin (EODC)
+
+- Intro Presentation and Questionnaire on who is the audience
+
+
+## 11:15 Finding Data, Running first graphs, difference to client-side processing
+
+Daniel (Sinergise)
+- Logging in People and showing them around
```

Comment:
Please change my section to:
___

- Client libraries (R & Python) 
- Connect to the Web Editor (login) and taking first processing steps
- Visualizing results through the Web Editor (incl. on-demand previews)

~~-> switch to python client here~~